bsp/nrf/lh2.c

```diff
@@ -1000,20 +1605,84 @@ void _spi_setup(const gpio_t *gpio_d) {
     NRF_SPIM->ENABLE = SPIM_ENABLE_ENABLE_Enabled << SPIM_ENABLE_ENABLE_Pos;
 }
 
+void _init_spi_ring_buffer(lh2_ring_buffer_t *cb) {
+    cb->writeIndex = 0;
+    cb->readIndex  = 0;
+    cb->count      = 0;
+
+    for (uint8_t i = 0; i < LH2_BUFFER_SIZE; i++) {
+        memset(cb->buffer[i], 0, SPI_BUFFER_SIZE);
+    }
```

Comment:
Cannot this be simplified by the following one liner?

```suggestion
    memset(cb, 0, sizeof(lh2_ring_buffer_t));
```

If that works, the corollary is now : do you need this function?

Reply:
you´re right, it's not needed, I removed it